ui/components/ui/unit-input/unit-input.component.js

```diff
@@ -106,6 +106,21 @@ export default class UnitInput extends PureComponent {
     this.props.onChange(value);
   };
 
+  handleOnKeyDown = (e) => {
+    const isNumericInput = DECIMAL_INPUT_REGEX.test(e.key)
```

Comment:
Can `this.keyPressRegex` be used here instead?

Reply:
it does not unfortunately